platform/broadcom/sonic-platform-modules-juniper/README.md

```diff
@@ -1 +1,291 @@
+```
+
+
+Note: Use the following command to build a demo target:
+
+```
+  $ make -j4 MACHINEROOT=../machine/juniper MACHINE=juniper_qfx5210 all demo
+```
+
+In addition to the above list of binary files, the following two binary files are also created:
+
+```
+-rw-rw---- 1 build build 12576008 Aug 19 16:30 demo-installer-x86_64-juniper_qfx5210-r0.bin
+-rw-rw---- 1 build build 12576008 Aug 19 16:30 demo-diag-installer-x86_64-juniper_qfx5210-r0.bin
+```
+
+You can install these binary files by using the 'onie-nos-install' command to test the install / boot workflow.
+
+Use the following command for make clean:
+
+```
+ $ make machine-clean MACHINEROOT=../machine/juniper MACHINE=juniper_qfx5210
+```
+
+
+## Installing ONIE on a New QFX5210-64C-S Switch
+
+The following information describes on how to install ONIE on the Juniper Networks QFX5210-64C-S switch. You can do a fresh install of ONIE image on the QFX5210-64C-S switch, or recover an existing ONIE image from the QFX5210-64C-S switch that has been corrupted. 
+
+To install ONIE on a new QFX5210-64C-S switch, you can use one of the following ONIE recovery images:
+
+1) .<machine>.iso       -- Hybrid ISO image.
+2) .<machine>.efi64.pxe -- PXE image for UEFI64 machines.
+
+
+## Creating an ISO Recovery Image
+
+You can use the recovery ISO (.iso) image to create a bootable USB memory device.
+
+To create a bootable USB memory device, use the "dd" command on a Linux workstation as follows:
+
+```
+ $ dd if=<machine>.iso of=/dev/sdX bs=10M
+```
+
+For example:
+
+```
+ $ dd if=onie-recovery-x86_64-juniper_qfx5210-r0.iso of=/dev/sdb bs=10M
+```
+
+You can find the correct "/dev/sdX" by validating the "dmesg" output after inserting an USB device into the Linux workstation.
+
+
+1) Booting from a USB Memory Device
+
+To boot from an external USB memory device connected to the QFX5210-64C-S switch, you need to:
+
+a. Insert the USB memory device to the USB port of the QFX5210-64C-S switch.
+
+b. Power on the QFX5210-64C-S switch and enter the BIOS configuration by pressing the Esc key, as displayed in the console screen.
+
+c. Set the hard drive boot order as follows:
+
+   When you see the "Boot Option #1" displayed on the console screen, select the USB memory device:
+
+    Boot-->Boot Option Priorities-->Boot Option #1
+
+   If the USB memory device name is not listed in "Boot Option #1", check the priorities in the hard drive boot order:
+
+    Boot-->Hard Drive BBS Priorities-->Boot Option #1
+
+
+   For example, consider "JetFlashTranscend 8GB 8.07" as the USB memory device, the boot order will display as follows:
+
+
+    Boot Option Priorities
+
+    Boot Option #1          [JetFlashTranscend 8...]
+
+    Boot Option #2          [ATP ATP IG eUSB 1100]
+
+    Boot Option #3          [IBA GE Slot 00A0 v1543]
+
+    Boot Option #4          [UEFI: Built-in EFI ...]
```

Comment:
Suggest to surround this in triple backticks to ensure it is displayed in monospaced font.